### PR TITLE
Visualization option enhancements

### LINKED ISSUE #1820: Style gauge visualization threshold options
The threshold options for a gauge visualization need some love:

screen shot 2018-12-10 at 2 48 57 pm

In particular, the color dropdowns should be updated to use the clockface components.

### LINKED ISSUE #1822: Enable decimal places visualization option
As user viewing the **gauge**, **single stat**, or **line graph + single stat** visualization options, I should be able to set the number of decimal places visible in the visualization.

### LINKED ISSUE #1823: Enable prefix and suffix gauge visualization options
As a user, I should be able to set the prefix and/or suffix that appears in a gauge visualization.

There's already UI elements for this in the gauge options, but they aren't hooked up.

### PR DESCRIPTION
**Pull request by chnn**
Closes #1822 
Closes #1820
Closes #1823

A few small enhancements to various visualization options. Additional details are in the commit messages.

### CODE REVIEW

File: ui/src/shared/components/ColorDropdown.scss [hunk 1]
```diff
@@ -1,119 +1,12 @@
...
-.color-dropdown--name {
-  position: absolute;
-  top: 50%;
-  transform: translateY(-50%);
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
 }
+
 .color-dropdown--swatch {
```
Comment (alexpaxton) [inline ui/src/shared/components/ColorDropdown.scss]: Soooo much simpler css

File: ui/src/clockface/components/auto_input/AutoInput.tsx [hunk 2]
```diff
@@ -101,32 +98,41 @@ export default class AutoInput extends Component<Props, State> {
   }
 
   private handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
-    const inputValue = Number(e.target.value)
+    const {max, min} = this.props
+
+    let inputValue = e.target.value
+
+    if (Number(inputValue) < min) {
```
Comment (alexpaxton) [inline ui/src/clockface/components/auto_input/AutoInput.tsx]: Ahh very nice and flexible

File: ui/src/shared/components/TimeMachineFluxEditor.tsx [hunk 3]
```diff
@@ -55,7 +55,7 @@ class TimeMachineFluxEditor extends PureComponent<Props> {
       },
       {
         render: () => <FluxFunctionsToolbar />,
-        handlePixels: 10,
+        handlePixels: 6,
```
Comment (alexpaxton) [inline ui/src/shared/components/TimeMachineFluxEditor.tsx]: I wonder if we should replace the `Threesizer` here with `DraggableResizer`

Comment (chnn) [reply ui/src/shared/components/TimeMachineFluxEditor.tsx]: Yea I think so... just kicking the can down the road here.

**Pull request merged**